### DISCUSSION TITLE
Schema evolution on upsert (merge)

### DISCUSSION THREAD

**Discussion by cesar-vermeulen** [delta-io/delta-rs#2281] [Q&A]
Hello!

With the awsome addition of merge schema support in the write operation (kudos contributors @ https://github.com/delta-io/delta-rs/pull/2246), I was wondering whether a similar functionality is on the roadmap for upsert transactions? Would be a great addition to the current functionalities!

Thanks,
Cheers!

**Comment by ion-elgreco** [comment]
@cesar-vermeulen We don't really have a roadmap actually, but happy to welcome contributions to also have schema evolution in MERGE operation : )
